### PR TITLE
jmap_calendar.c: properly handle participantReply with a plus address

### PR DESCRIPTION
**Pull request by ksmurchison**

### CODE REVIEW

**Comment by wolfsage** [comment]
This seems right to me!

File: cassandane/tiny-tests/JMAPCalendars/calendarevent_participantreply_plusaddr [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    ]);
+
+    my $participants = {
+        "org" => {
+            "name" => "Cassandane",
+            roles => {
+                'owner' => JSON::true,
+            },
+            sendTo => {
+                imip => 'cassandane@example.com',
```
Comment (rsto) [inline cassandane/tiny-tests/JMAPCalendars/calendarevent_participantreply_plusaddr:33]: This must be a mailto: URI

**Review by rsto — APPROVED** [review]
Looks good except for one nit.

File: cassandane/tiny-tests/JMAPCalendars/calendarevent_participantreply_plusaddr (hunk 1 above)
Comment (ksmurchison) [reply cassandane/tiny-tests/JMAPCalendars/calendarevent_participantreply_plusaddr]: Fixed

**Pull request merged**